docs/modules/ROOT/pages/wan-replication.adoc

```diff
@@ -1,54 +1,116 @@
-= Synchronizing Clusters
-:description: To synchronize data between two or more clusters, you can use WAN Replication.
+= Synchronizing Data Between Clusters
```

Comment:
"Synchronize" keyword is not something known in this industry, they mostly use "Data Replication", some samples;

https://buildatscale.tech/redis-replication/
https://www.cockroachlabs.com/docs/stable/demo-replication-and-rebalancing.html
https://www.toptal.com/mysql/mysql-master-slave-replication-tutorial
https://university.scylladb.com/courses/scylla-essentials-overview/lessons/architecture/topic/data-replication/

Reply:
I've changed the heading and nav item to reflect this, and a couple of instances in the text.